src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/CancelVideoBookingService.kt

```diff
@@ -12,13 +14,25 @@ import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository.VideoBooki
  * [VideoBooking]'s which have already taken place cannot be cancelled.
  */
 @Service
-class CancelVideoBookingService(private val videoBookingRepository: VideoBookingRepository) {
+class CancelVideoBookingService(
+  private val videoBookingRepository: VideoBookingRepository,
+  private val bookingHistoryService: BookingHistoryService,
+) {
+  companion object {
+    private val log = LoggerFactory.getLogger(this::class.java)
+  }
+
   @Transactional
   fun cancel(videoBookingId: Long, cancelledBy: String): VideoBooking {
     val booking = videoBookingRepository
       .findById(videoBookingId)
       .orElseThrow { EntityNotFoundException("Video booking with ID $videoBookingId not found.") }
 
-    return videoBookingRepository.saveAndFlush(booking.cancel(cancelledBy))
+    // TODO: Need to raise event to deal with cancelled or removing the remote appointments in A&A or NOMIS
```

Comment:
TODO not necessary.  Event is being raised.

Reply:
Is it? Can't see where?

Reply:
I was expecting it to be in this service method, but its not.

Reply:
BOOKING FACADE. Duh.

Reply:
Removed TODO